script_library/bootstrap-ansible-if-necessary.sh

```diff
@@ -6,16 +6,16 @@ set -e
 # is packaged for all the distributions.
 
 function install_ansible (){
-    # NOTE(toabctl): ${SOCOK8S_WORKSPACE_BASEDIR} and ${SOCOK8S_ENVNAME} are always set
-    local socok8s_workspace=${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace
-    if [[ ! -d ${socok8s_workspace}/.ansiblevenv/ ]]; then
-        virtualenv ${socok8s_workspace}/.ansiblevenv/
-        source ${socok8s_workspace}/.ansiblevenv/bin/activate
-        pip install --upgrade pip
-        pip install --upgrade -r $(dirname "$0")/script_library/requirements.txt
-        python -m ara.setup.env > ${socok8s_workspace}/.ansiblevenv/ara.rc
+    # NOTE(toabctl): ${SOCOK8S_TMP_BASEDIR} and ${SOCOK8S_ENVNAME} are always set
+    local socok8s_venvspace=$SOCOK8S_TMP_BASEDIR/${SOCOK8S_ENVNAME}-cache
+    if [[ ! -d ${socok8s_venvspace}/.ansiblevenv/ ]]; then
+        virtualenv ${socok8s_venvspace}/.ansiblevenv/
```

Comment:
virtualenv here could point to python2 virtualenv, if we want to force this to be a python3 venv (which seems to me based on line 14 use of pip3) then we should pass the `--python=python3` flag here